faas

```diff
@@ -3,25 +3,34 @@
 function print_usage() {
 cat <<EOF
 
-USAGE: faas create -f <function> -p <path> -v <version>
-  This command will build the function based on the code available in the path directory
+USAGE: 
 
-Flags:
+faas build -n <name> -v <version> -f <path>
+  This command will build the function based on the code available in the path directory, using the name 
+  specified for the image that is built
 
-[*] -f  | --function - the name of the function
-    -p  | --path - the path to the function code (defaults to current directory)
-    -v  | --version - the version of the function created (defaults to v0001)
+faas create -f <path>
+  This command will create the resource[s] based on the resource definition[s] that the path points to
 
-[*] = Required arguments
+Options:
```

Comment:
shouldn't `-n` and `-v` only be supported when using `faas build`?

it seems like we need separate usage... the top level should just show sub-commands, and then for specific usage, we'd support something like `faas <command> -h`

Reply:
Yeah, we could do that. Let me add a commit